egs/librispeech/s5/local/chain/run_tdnn_discriminative.sh

```diff
@@ -96,7 +96,7 @@ if [ $frame_subsampling_factor -ne 1 ]; then
     data_dirs=
     for x in `seq -$[frame_subsampling_factor/2] $[frame_subsampling_factor/2]`; do
       steps/shift_feats.sh --cmd "$train_cmd --max-jobs-run 40" --nj 350 \
```

Comment:
This is not really the fix I wanted, sorry.. I think you should remove that arg (mfcc_hires) and change steps/shift_feats.sh (which is deprecated) to utils/data/shift_feats.sh.

Reply:
Ahha, got it.